cloudformation_templates/alarms/HTTP5xxCountAlarm.j2

```diff
@@ -0,0 +1,5 @@
+{% extends "alarms/base_alarm.j2" %}
+
+{% block description %}Login failure rate too high{% endblock %}
```

Comment:
The file is HTTP5xx alarm but the message is about login failures - doesn't seem right?